app/assets/stylesheets/image_x.scss

```diff
@@ -14,6 +14,23 @@ $image-thumb-slider-background-color: $sul-background;
   position: relative;
 }
 
+// This is used for the fullheight option
+// This gives the image viewer enough
+// space given the header/footer
+.#{$namespace}-fullheight {
+  .#{$namespace}-image-x {
+    height: calc(100vh - 120px);
+  }
+
+  // When in the fullheight mode we
+  // want to hide the full-screen button
```

Comment:
What you're doing here seems fair enough to me, but is this tested anywhere? I assume it would be a fairly simple spec to check that the button is indeed hidden from view.